server/admin-cli.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
```

Comment:
Is this file executable? It should have the same permissions as the other scripts.

Also, could you move it to the bin folder?

Reply:
It has the same permissions as all the other scripts in `bin`.